src/org/ods/util/HelmStatusSimpleData.groovy

```diff
@@ -0,0 +1,186 @@
+package org.ods.util
+
+import com.cloudbees.groovy.cps.NonCPS
+import groovy.transform.TypeChecked
+
+@TypeChecked
+class HelmStatusSimpleData {
+
+    String name
+    String version
+    String namespace
+    String status
+    String description
+    String lastDeployed
+    /**
+     * Resources names by kind
+     */
+    Map<String, List<String> > resourcesByKind
+
+    @SuppressWarnings(['Instanceof'])
+    @NonCPS
+        static HelmStatusSimpleData fromJsonObject(Object object) {
```

Comment:
This is a Map. It's irrelevant if it comes from a Json or not. We can rename it to just from (taking explicitly a Map)

Reply:
For this case, according to the current implementation, object being a Map type is not guaranteed, see: https://github.com/opendevstack/ods-jenkins-shared-library/pull/1143#discussion_r1806245248